src/client/src/app/global.css

```diff
@@ -236,3 +236,10 @@
 .nhsuk-tabs__list-item::before {
   content: none;
 }
+
+.feedback-banner {
+  text-align: center;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  background-color: #fff;
+}
```

Comment:
This styling doesn't look quite right to me - on the designs in the ticket the banner tag/text starts at the same breakpoint as the rest of the site content. Because of `text-align: center` it's appearing out of line:
<img width="788" alt="image" src="https://github.com/user-attachments/assets/b16b7dcf-a6bd-4671-8957-0b005d92dd27" />